SIS_import/sis_set_url.rb

```diff
@@ -232,13 +251,18 @@ def ESB_APICall(url, authorization_string, content_type, request_type, param_has
 			# if parameter hash is not null, attach them to form
 			request.set_form_data(param_hash)
 		end
+
+		p param_hash
```

Comment:
The STDOUT print will be included in the cron job email. So it is easy to glance the email than to log into caprice or monza server to see the log file.